core/controllers/topic_editor.py

```diff
@@ -585,7 +613,14 @@ class TopicRightsHandler(
 
     @acl_decorators.can_view_any_topic_editor
     def get(self, topic_id: str) -> None:
-        """Returns the TopicRights object of a topic."""
+        """Fetches the topic rights of a topic.
+
+        Args:
+            topic_id: str. The ID of the topic.
+
+        Raises:
+            InvalidInputException. Expected a valid topic id to be provided.
```

Comment:
InvalidInputException: The topic ID provided is not valid.

Reply:
I have made the changes.